src/main/java/com/alexnederlof/jasperreport/CompileTask.java

```diff
@@ -55,24 +55,13 @@ public CompileTask(File source, File destination, Log log, boolean verbose) {
      */
```

Comment:
Probably very nitpick, but in the javadoc here it says that this method will 

> skip the file if it doesn't have the right extension
Does it still makes sense? Why don't just remove this "documentation" at all.

Reply:
It doesn't – removed.